spec/features/user/02_user_sign_in_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+feature 'sign in', %{
+  As an unauthenticated user
+  I want to sign in
+  So that I can post items and review them
+  } do
+    # ACCEPTANCE CRITERIA
+    # * I must enter a user's email address
+    # * I must enter a user's password
+    # * If I do not perform the above, I get an error message
+    # * If I specify valid information, I sign in to my account and am authenticated
```

Comment:
Line is too long. [84/80]<br>Incorrect indentation detected (column 4 instead of 2).